std/regex/package.d

```diff
@@ -320,13 +320,11 @@ public alias StaticRegex(Char) = std.regex.internal.ir.StaticRegex!(Char);
 
     Throws: $(D RegexException) if there were any errors during compilation.
 +/
```

Comment:
This doc comment needs to be moved down to `regex()`; now it refers to the newly inserted `regexPure()`.